cypress/integration/namespaces.spec.js

```diff
@@ -74,12 +74,12 @@ describe('Namespace Select', () => {
 
   it('have the correct namespaces in the dropdown when using navigation header', () => {
     cy.get('@namespace-select-button').click({ wait: 1000 });
-    cy.get('.prose > .text-2xl').contains('Select a Namespace');
+    cy.get('h2').contains('Select a Namespace');
```

Comment:
If we put another `<h2>` on this page, this test will break. I had something in mind more like adding a `data-cy` attribute on the h2 element, then using that in the test.

Something like `cy.get('[data-cy="namespace-selector-heading"]').contains('Select a Namespace');`